Doppler.AccountPlans/Model/PlanRenewalInformation.cs

```diff
@@ -0,0 +1,8 @@
+namespace Doppler.AccountPlans.Model
+{
+    public class PlanRenewalInformation
```

Comment:
Maybe the name of class should be ```PlanDiscountInformation``` becuase the information is about the discounts of the plan. What do you think?